modules/troubleshooting/pages/builds.adoc

```diff
@@ -121,7 +121,7 @@ for old_ref in ${OLD_REFS}; do
     [[ $new_ref == $old_ref ]] && continue
     echo "New digest found! $new_ref"
     for file in $FILES; do
-        sed -i -e "s!${old_ref}!${new_ref}!g" $file
+        pmt migrate --new-bundle "$new_ref" --pipeline-file "$file" <1>
     done
```

Comment:
Would it be more efficient if we ran `pmt migrate` on all the files at the same time, instead of doing it one by one? I imagine we would only have to do one round of migrations resolution in that case, which could make a significant difference for repos with many pipelines